lib/statesman/adapters/active_record.rb

```diff
@@ -87,6 +87,12 @@ def transitions_for_parent
       end
 
       def unset_old_most_recent
+        most_recent = transitions_for_parent.where(most_recent: true)
+        updated_at = if ActiveRecord::Base.default_timezone == :utc
```

Comment:
Worth noting to other reviewers that this is how [activerecord does timestamps](https://github.com/rails/rails/blob/5-1-stable/activerecord/lib/active_record/timestamp.rb#L78). I checked incase we where going to miss some strange timezone stuff.

Reply:
Yup, it's a shame its not a public method - would be useful when using `update_all`